src/test/mir-opt/instrument_coverage.rs

```diff
@@ -3,7 +3,7 @@
 // intrinsics, during codegen.
 
 // needs-profiler-support
-// compile-flags: -Zinstrument-coverage
+// compile-flags: -Zinstrument-coverage --remap-path-prefix={{cwd}}=/the/cwd
```

Comment:
@wesleywiser @tmandry 
See this change, and the resulting path changes in the mir-opt diffs ([for example](https://github.com/rust-lang/rust/pull/75037/files#diff-55fcf1fc571847cc4126002d65e38725)).

This ensures the byte array representations are also the same, no matter where the absolute build paths are.